add new notebook for hands on with gilda and bioregistry

tonywu1999 opened the pull request:
Hey Ben, would be happy if you could provide any additional feedback to this PR if you have time.

## Changes
- Added new python notebook introducing how to use gilda and bioregistry in the context of a dataset produced from MSstats output. Notably:
    - I grounded protein IDs and drug names from the dataset
    - I introduced how to apply context to gilda groundings
    - I introduced how bioregistry can normalize curies for uniprot and determine if a uniprot ID is valid

## Notable Issues
- I tried to ground ATR based on context of other less likely results (like ANTXR1), but results still ended up the same as before.
- Not all drugs could be grounded on gilda, but I grounded 2 of them as of now.

Commit:
add new notebook for hands on with gilda and bioregistry

bgyori commented:
Thanks @tonywu1999! One issue is that in the data set, proteins are identified by UniProt mnemonic (ZZZ3_HUMAN) which is actually a form of identifier rather than text name. So grounding these using Gilda (which takes a text name as input and produces an identifier as output) is not quite right. So we don't need to demo or discuss this.

For the mnemonics, instead of grounding one would apply ID conversion using INDRA to get a proper UniProt ID:
```python
In [1]: from indra.databases import uniprot_client

In [2]: uniprot_client.get_id_from_mnemonic('ZZZ3_HUMAN')
Out[2]: 'Q8IYH5'
```
Grounding in this data set is best demonstrated on the drug names or on the informal drug target names which you already have.

In terms of ATR an disambiguation: "ATR" is not among the strings for which there is a disambiguation model in Gilda so we wouldn't expect to see interesting results here - no need to demo this.

Commit:
Simplified gilda/bioregistry notebook by removing references to gilda context and removing groundings on protein IDs

Commit:
add an example to normalize curie for a chebi ID

tonywu1999 commented:
@bgyori 

Thanks for the feedback Ben.  I removed the part with Uniprot mnenomic ID transformation and context disambiguation for now.  I also added an additional example of using bioregistry to normalize a curie.

Commit:
add grounding for one of the drug targets

Commit:
fix of chk1 grounding code

Commit:
rename gilda bioregistry file
